.buildkite/windows.rayci.yaml

```diff
@@ -1,5 +1,11 @@
 group: windows
 steps:
+  - name: hello-windows
+    wanda: .buildkite/windows.wanda.yaml
+    instance_type: builder-windows
+    env:
+      DOCKER_BUILDKIT: "0"
```

Comment:
this won't really have an effect. when wanda runs docker, it does not inherit from the environment.

can we upgrade the windows docker to support buildkit?

Reply:
> can we upgrade the windows docker to support buildkit?

I'm not sure how too; what is the time of effort required to do this?